drivers/clock_control/clock_control_litex.c

```diff
@@ -975,16 +975,16 @@ static int litex_clk_set_duty_cycle(struct litex_clk_clkout *lcko,
 	   *low_time = &lcko->div.low_time;
 
 	if (lcko->frac.frac == 0) {
-		int ret;
+		int calc_ret;
```

Comment:
It would seem this re-definition is unnecessary. Just removing line 978 would seem better

Reply:
Removed.